ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-master.yaml

```diff
@@ -67,7 +67,7 @@ images:
 operator:
   bundles:
   - dockerfile_path: openshift-ci/Dockerfile.bundle.ci
-  - dockerfile_path: openshift-ci/Dockerfile.bundle-4.9.ci
+  - dockerfile_path: openshift-ci/Dockerfile.bundle-4.10.ci
```

Comment:
we need a PAO PR to create this, and make this PR depend on that one, don't we?

Reply:
in general, I already have PR that creates the relevant file, we should just merge one of PRs to make it work.

Reply:
Well, thing is I'm not sure at all the CI for this PR will pass without this file being already added (point in case: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/25865/rehearse-25865-pull-ci-openshift-kni-performance-addon-operators-master-ci-index/1488475991889154048)

Reply:
Yep, it will not, but it is not a required job, and once we merge this one, the PAO one should pass https://github.com/openshift-kni/performance-addon-operators/pull/834.

Reply:
ok, let's move forward